pkg/api/defaults.go

```diff
@@ -51,7 +51,8 @@ const FLAG_FALSE string = "false"
 var DefaultOperatorFlags OperatorFlags = OperatorFlags{
 	"aro.alertwebhook.enabled": FLAG_TRUE,
 
-	"aro.azuresubnets.enabled": FLAG_TRUE,
+	"aro.azuresubnets.nsg.enabled":             FLAG_TRUE,
```

Comment:
This will only enable it for new clusters -- to enable the change on existing clusters we will need to pass these flags in as a part of the PUCM.

Reply:
Correct.